### DISCUSSION TITLE
Is there no way to clean mods I selected by keyboard?

### DISCUSSION THREAD

**Discussion by AB-SnowyNight** [ppy/osu#29593] [Ideas]
I can't clean mods by press key 1 like stable, is it worth to add in osu!lazer?

**Comment by Gabixel** [comment]
<kbd>Backspace</kbd> is probably what you're looking for. It removes all the mods you selected.

**Reply by Gabixel** [reply]
It's also present in the keybindings list:

![image](https://github.com/user-attachments/assets/63770ecc-5eee-4dd5-9e4d-ef7d13caf038)

**Reply by vatei** [reply]
![image](https://github.com/user-attachments/assets/b884339b-daee-47d6-928f-4cb256603bf7)
This setting should also probably be disabled if you don't want to add a click
